MerlinAU.sh

```diff
@@ -4797,6 +4811,95 @@ _ShowNodesMenu_()
    printf "${SEPstr}"
 }
 
+_ShowNodesMenuOptions_()
+{
+    while true
+    do
+        _ShowNodesMenu_
+        printf "\nEnter selection:  "
+        read -r nodesChoice
+        echo
+        case $nodesChoice in
+            e|exit) break
+               ;;
+            *) _InvalidMenuSelection_
+               ;;
+        esac
+    done
+}
+
+##---------------------------------------##
+## Added by ExtremeFiretop [2024-May-02] ##
+##---------------------------------------##
+_DownloadChangelogs_()
+{
+    changeLogTag="$(echo "$(nvram get buildno)" | grep -qE "^386[.]" && echo "386" || echo "NG")"
+    if [ "$changeLogTag" = "386" ]
+    then
+        wget -O "$FW_BIN_DIR/Changelog-${changeLogTag}.txt" "${CL_URL_386}"
+    elif [ "$changeLogTag" = "NG" ]
+    then
+        wget -O "$FW_BIN_DIR/Changelog-${changeLogTag}.txt" "${CL_URL_NG}"
+    fi
+    changeLogFile="${FW_BIN_DIR}/Changelog-${changeLogTag}.txt"
+    if [ ! -f "$changeLogFile" ]
+    then
+        Say "Change-log file [$changeLogFile] does NOT exist."
+    else
+        clear
+        printf "\n${GRNct}Changelog is ready to review!${NOct}\n"
+        printf "\nPress '${REDct}q${NOct}' to quit when finished.\n"
+        _WaitForEnterKey_
+        less "$changeLogFile"
```

Comment:
I recommend using the "`more`" command instead of `less` to avoid showing the <CR-LF> end of lines:
`more "$changeLogFile"`

This is using `less` cmd:
![ChangeLog_ViewWithLess](https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/assets/119833648/a6e65224-96e7-4f8f-a2ff-f444b6a19ee0)

This is using `more` cmd:
![ChangeLog_ViewWithMore](https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/assets/119833648/c3fabb6a-cc94-439b-be84-3e812f0ba0e4)

Reply:
Good suggestion, I can do that. :)

Reply:
@Martinski4GitHub 

Fixed.